qiling/os/windows/structs.py

```diff
@@ -115,6 +117,35 @@ class PEB(Struct):
 
     return PEB
 
+class NT_TIB(struct.BaseStruct):
```

Comment:
Here and on other new structures:
Are you sure the kd output is needed here? If possible, please refer to a URL where the sturcutre layout can be found rather than pasting it here.

Reply:
Yes, I am aware.. if there is an unformal documentation you can link to, that's perfectly fine.